dali/base/dadfs.cpp

```diff
@@ -1243,13 +1243,11 @@ static void setUserDescriptor(Linked<IUserDescriptor> &udesc,IUserDescriptor *us
     udesc.set(user);
 }
 
+static bool scopePermissionsAvail = true;
```

Comment:
Agreed with the comment on the preexisting C-style pattern throughout the file...